sssom/util.py

```diff
@@ -440,13 +461,19 @@ def dataframe_to_ptable(df: pd.DataFrame, priors=None, inverse_factor: float = 0
 RDF_FORMATS = {"ttl", "turtle", "nt", "xml"}
 
 
-def sha256sum(filename):
+def sha256sum(filename: str) -> str:
+    """Hashing function.
```

Comment:
Here's a more informative, better written docstring:

```suggestion
    """Calculate the SHA256 hash over the bytes in a file.
```